packages/rocketchat-ui-flextab/flex-tab/tabs/membersList.coffee

```diff
@@ -57,7 +57,7 @@ Template.membersList.helpers
 	canAddUser: ->
 		roomData = Session.get('roomData' + this._id)
 		return '' unless roomData
-		return roomData.t in ['p', 'c'] and roomData.u?._id is Meteor.userId()
+		return roomData.t in ['p', 'c'] and RoomModeratorsAndOwners.findOne({ rid: this._id, 'u._id': Meteor.userId(), roles: { $in: ['moderator','owner'] }})
```

Comment:
Instead of assuming owners and moderators can add users, let's create a permission for adding users to channels. Then we can assign that permission to other roles with ease.